codesystems/CodeSystem-UKCore-MedicationPrescribingOrganizationType.xml

```diff
@@ -0,0 +1,40 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
```

Comment:
The CodeSystem wasn't specifically mentioned in item 14 or 15, I did originally change this in a different branch to the ones I'd did for the extension and the valueset, but I think we then discussed it and we'd decided to leave it as it was.  I think it would be good to get @johngeo opinion on this.